docs/release_notes.md

```diff
@@ -34,6 +36,11 @@
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
 
+### Known Limitations
```

Comment:
Should we also include at "Bug Fixes" #1198? Despite we initialize created for AdnSkinMerge we applied to all deformers.

Reply:
It's already added, we would have to generalize it for all deformers.